content/en/universal_service_monitoring/_index.md

```diff
@@ -315,6 +321,7 @@ Add the following to your `docker run` command:
 --cap-add=NET_RAW \
 --cap-add=IPC_LOCK \
 --cap-add=CHOWN
```

Comment:
```suggestion
--cap-add=CHOWN \
```

Reply:
Not sure if this is just a misunderstanding on my part, but it looks like this might need to be added since an extra line is being added below. Feel free to ignore this suggestion if this is just my own misunderstanding.